sunkit_image/utils/tests/test_noise.py

```diff
@@ -1,36 +1,38 @@
 import numpy as np
 from skimage import data
 
-from ..noise import NoiseLevelEstimation as nle
+import sunkit_image.utils.noise as nf
 
 img = data.camera()
 
 
-def convmtx2_test():
+def convmtx2_test(img):
```

Comment:
I think our convention is the other way around: 
```suggestion
def test_convmtx2(img):
```

Reply:
An this test should probably be called "test that the output size is correct", not the implementation. Otherwise we may think we are testing it all.